tests/phpunit/data/html-api/html-xpath-generating-processor.php

```diff
@@ -9,48 +9,16 @@ class HTML_XPath_Generating_Processor extends WP_HTML_Processor {
 	 */
```

Comment:
This class and filename are now inaccurate, since xpath was removed. Something about "token counting" perhaps?

Reply:
Right on. Done in 7f39293